src/components/Placeholders/PlaceholderThemesListModal.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { useHistory } from 'react-router-dom'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import PlaceholderThemesList from 'src/components/Placeholders/PlaceholderThemesList'
+
+const PlaceholderThemesListModal = () => {
+  const { t } = useI18n()
+  const history = useHistory()
+
+  return (
+    <PlaceholderThemesList
+      title={t('PlaceholdersList.title', { name: '' })}
+      onClose={history.goBack}
+    />
+  )
+}
+
+export default PlaceholderThemesListModal
```

Comment:
nit: I don't see the point of such a small component for now.

Do you plan to add more feature inside? if not, let's inline it

Reply:
c'est un comp qui sert de "vue", une route pointe donc sur lui, à notre sens c'est plus simple d'avoir cette approche